src/rajAJun20/ReverseNumber.java

```diff
@@ -0,0 +1,20 @@
+package rajAJun20;
+
+public class ReverseNumber {
+	
+	void printReverseNumber(int inputNumber) {
+		while(inputNumber > 0) {
+			int reverse = inputNumber % 10;
+			inputNumber = inputNumber/10;
+			System.out.println("Reverse number is : " + reverse);
```

Comment:
This will not work. 
void findReverseNumber(int number) {
		int reverseNum = 0;
		int mod =0;
		while(number>0) {
			mod = number%10;
			number = number/10;
		    reverseNum = mod + reverseNum*10 ;
		}
		System.out.println("Reverse Number is "+reverseNum);
	}